src/components/AuthForm.js

```diff
@@ -188,14 +188,41 @@ const authSchemas = {
         password: {"ui:widget": "password"}
       }
     }
-  }
+  },
+  portier: {
+    schema: {
+      ...baseAuthSchema,
+      required: [...baseAuthSchema.required, "email"],
+      properties: {
+        ...baseAuthSchema.properties,
+        email: {
+          title: "Email address",
+          type: "string",
+          format: "email",
+        }
+      }
+    },
+    uiSchema: {
+      authType: {
+        ...baseUISchema.authType,
+        "ui:help": (
+          <span>
+            <b>Note:</b> The
+            <a href="https://github.com/Kinto/kinto-portier">{" kinto-portier "}</a>
```

Comment:
because I'm still feeling bad about that I've asked on twitter about it https://twitter.com/n1k0/status/840256319638654976

Not sure I'll feel better but TGIF.